examples/00-load/linear-cells.py

```diff
@@ -0,0 +1,200 @@
+cell. Non-linear cells contain additional points along the edges of the cell.
+
+For more details regarding what a :class:`pyvista.UnstructuredGrid` is, please
+see :ref:`point_sets_api`.
+
+"""
+import numpy as np
+
+import pyvista as pv
+from pyvista.examples import cells as example_cells, plot_cell
+
+# random generator for examples
+rng = np.random.default_rng(2)
+
+
+###############################################################################
+# Plot an example cell
+# ~~~~~~~~~~~~~~~~~~~~
+# PyVista contains a simple utility to plot a single cell, which is the
+# fundamental unit of each :class:`pyvista.UnstructuredGrid`. For example,
+# let's plot a simple :func:`Wedge <pyvista.cells.Wedge`.
+#
+grid = example_cells.Wedge()
+example_cells.plot_cell(grid)
+
+
+###############################################################################
+# This linear cell is composed of 6 points.
+
+grid.points
+
+
+###############################################################################
+# The UnstructuredGrid is also composed of a single cell and the point indices
+# of that cell are defined in :attr:`cells <pyvista.UnstructuredGrid.cells>`.
+#
+# .. note::
+#    The leading ``6`` is the number of points in the cell.
+
+grid.cells
+
+
+###############################################################################
+# Combine two UnstructuredGrids
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# We can combine two unstructured grids to create a single unstructured grid
+# using the ``+`` operator.
+#
+# .. note::
+#    This is an inefficient way of creating :class:`UnstructuredGrid`
+#    objects. To see a more efficient implement ion see
+#    :ref:`create_unstructured_example`.
+
+grid_a = example_cells.Hexahedron()
+grid_a.points += [0, 2.5, 0]
+
+grid_b = example_cells.HexagonalPrism()
+
+combined = grid_b + grid_a
+
+plot_cell(combined, cpos='iso')
+
+
+###############################################################################
+# This example helps to illustrate meaning behind the :attr:`cells
+# <pyvista.UnstructuredGrid.cells>` attribute. The first cell, a hexahedron
+# contains 8 points and the hexagonal prism contains 12 points. The ``cells``
+# attribute shows this along with indices composing each cell.
+
+combined.cells
+
+
+###############################################################################
+# Cell Types
+# ~~~~~~~~~~
+# PyVista contains the :class:`pv.CellType` enumerator, which contains all the
+# available VTK cell types mapped to a Python enumerator. These cell types are
+# used when creating cells and also can be used when checking the
+# :attr:`celltypes <pyvista.UnstructuredGrid.celltypes>` attribute. For example
+# ``combined.celltypes`` contains both the ``pv.CellType.HEXAHEDRON`` and
+# ``pv.CellType.HEXAGONAL_PRISM`` cell types.
+
+print(pv.CellType.HEXAHEDRON, pv.CellType.HEXAGONAL_PRISM)
+(pv.CellType.HEXAHEDRON, pv.CellType.HEXAGONAL_PRISM) == combined.celltypes
```

Comment:
The statement seems to have no effect.
```suggestion
assert (pv.CellType.HEXAHEDRON, pv.CellType.HEXAGONAL_PRISM) == combined.celltypes
```

Reply:
Doesn't it print `True` via `repr`?

Reply:
Yep.  It's only just supported to show it's the same.

I think we should keep `assert` statements to our testing.

Reply:
Sorry. This is an EXAMPLE, not a test. I misunderstood.